### ISSUE TITLE
Multiple open windows now overwrite each other

### ISSUE THREAD

**Issue by glenjamin** [excalidraw/excalidraw#4837]
Previously I was able to open multiple excalidraw windows with a different diagram in each one, but recently they all seem to sync to the same contents

Was this an intentional change, and is there any way to opt out of it?

**Comment by ad1992** [comment]
Yes, this was an [intentional change](https://github.com/excalidraw/excalidraw/pull/4545) so that all tabs sync the data, and users never end up losing drawings due to getting overridden by the latest one. Earlier if there were multiple tabs, each with a separate drawing, and if you close a tab and reopen again (or open a new tab), it will contain last state from the other tab and no way to recover the lost drawing.

**Comment by glenjamin** [comment]
Oh I see.

The observed behaviour when attempting to open a scene in a new tab when there's already one open is quite surprising, as it simply opens the drawing from the existing tab

It might be good to display a warning about what's going on, or even to provide the option to open it anyway with the localStorage sync disabled

In my experience I have relied far more on the behaviour of independent tabs than I have on the ability to close and re-open a tab. Generally my mental model has been that closing a tab will discard a scene (even though it doesn't) and I'm always slightly surprised when I go to start a new diagram and it resumes from whatever I worked on before

**Comment by alondrasmt89** [comment]
Hello:

Having a the same issue and I have lost the work for a week and have. Their is no way to recover the work? I´m trying to used the ctrl+z command, but it is not working. It is like their no history at all.

Thanks,

Alondra

**Comment by ad1992** [comment]
We are really sorry but there is no way to recover those drawings since the tabs are now always synced so if you had different drawings in different tabs, it would now show the latest one in all tabs. As mentioned in https://github.com/excalidraw/excalidraw/issues/4837#issuecomment-1048479695 we started syncing tabs as users were losing work. Please export your drawings If you want to keep a copy of them. Since we don't support tabs so if you want to work on multiple drawings at the same time the only way would be to open it in different browsers/ incognito might work too.

**Comment by svenn71** [comment]
1. open a tab, go to excalidraw.com, drag-drop an excalidraw file into the canvas
-> the excalidraw drawing is shown, as is the file name for saving
2. open another tab, go to excalidraw.com, drag-drop an another excalidraw file into the canvas
-> the excalidraw drawing is shown, as is the file name for saving
3. Jump back and forth between the two tabs, the sync mechanism make the file save feature go away
4. Save file, the suggested filename is now Untitled-2022-xxxxx
-> The filename of the original drawing is lost

I think the mentioned PR introduced something with this sync feature which needs to be adjusted.

**Comment by momer** [comment]
Another work-around is to use [Firefox Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/) (rather than incognito/private browsing).

**Comment by OliverRC** [comment]
How do you work on multiple documents in parallel ?

**Comment by marcospgp** [comment]
It seems like it's been a couple years, is a fix for this planned?

**Comment by Mrazator** [comment]
As it was explained by @ad1992 at some point this was an intentional change. Since then this is how https://excalidraw.com/ behaves. Our premium offering offers to store multiple collections in the cloud https://plus.excalidraw.com/.

We have a "tab" support in the backlog of https://excalidraw.com/, but it was not yet implemented.

Closing in favor https://github.com/excalidraw/excalidraw/issues/1543.

**Comment by marcospgp** [comment]
@Mrazator I believe you guys should make it clearer that having multiple projects open at once is a benefit of the premium plan. I did end up signing up for it, but I may have done so earlier if I knew the issue was an intended free plan limitation.

**Comment by OliverRC** [comment]
@Mrazator you make it sound like this is obvious and they we should somehow know it is a premium feature.
I agree with @marcospgp that it really could be made clearer.

I mean where on your pricing and feature comparisons is it made obvious that having multiple drawings open is a premium feature

![Image](https://github.com/user-attachments/assets/ff53ea80-b9ce-4883-954f-acc30d0d9936)
![Image](https://github.com/user-attachments/assets/58e573bd-0b1f-45f0-b3f0-f204024a1425)

**Comment by Mrazator** [comment]
> I believe you guys should make it clearer that having multiple projects open at once is a benefit of the premium plan.

It's mentioned in various places on our landing page, i.e. https://plus.excalidraw.com/pricing/. Though I agree it does not completely scream this information.

![Image](https://github.com/user-attachments/assets/f7ebf969-a0ca-4933-9d12-238bc76c566f)

It might make sense to add it somewhere inside the editor (i.e. when trying to create a new scene / clean canvas), though we are trying not to upsell the premium plan in the editor too much as it creates additional friction - the open-source version, even with the one opened scene workflow, is perfectly fine for a lot of users.

> I did end up signing up for it, but I may have done so earlier if I knew the issue was an intended free plan limitation.

Awesome! Though to clarify, I think the intention was mainly to limit the loss of data, rather than to give the premium an advantage. Assuming we ended up with this as the only feasible solution, as attacking "tabs" without the intermediate step would be too much for one change.

**Comment by marcospgp** [comment]
nobody will ever want to open the same project in two different tabs, so why not display the information that multiple projects at once is a premium feature on that new tab?
